cvmfs/server/cvmfs_server_json.sh

```diff
@@ -91,6 +91,11 @@ _render_info_file() {
     echo '  "last_geodb_update" : "'$modtime'",'
   fi
 
+  if [ "x${CVMFS_PUBLISH_VERSIONS_IN_META_FILE}" == "xtrue" ]; then
+    echo '  "cvmfs_version" : "$(cvmfs_version_string)",'
+    echo '  "os_version" : "' $(lsb_release -d | cut -f2) '", '
```

Comment:
For this we'd need to add the `lsb-release` dependency to the cvmfs-server package.

Reply:
And watch out, "there are no plans to ship lsb-release in RHEL9".  Should use /etc/os-release instead.

Reply:
At some point I will understand what the community decides on as a dependable, long term, consistent way of figuring out the running version of linux... I'm not sure we can depend on `/etc/os-release` as it is theoretically tied to systemd, and as much as Debians and RHEL clones have it, god knows for how long? I am happy to use it though if the CVMFS community prefers it they way, but it should probably be moved to a function somewhere ala `_get_os_description` or similar.

Other options is to drop the data, or doing `uname -r`, but `uname` leaks data we probably don't want to announce and doesn't really give me what I want anyway. Hm. I'm very open to ideas here.

Reply:
With regards to `/etc/os-release`, the default way to parse the file is to source it, ala:
````
test -e /etc/os-release && os_release='/etc/os-release' || os_release='/usr/lib/os-release'
. "${os_release}"
````
However, this pollutes the environment of the running script with environment variables like `NAME`, `ID`, `HOME_URL`, and more. The environment variables used in `cvmfs_server` are all sanely prefixed and should avoid being clobbered, but I'm still a bit nervous about doing a raw source late in the scripts run time. 

My thought would be to either parse the file, or if sourcing it, doing it very early in https://github.com/cvmfs/cvmfs/blob/devel/cvmfs/server/cvmfs_server_coda.sh to ensure any clobbering is done by CMVFS code and not from `/etc/os-release`.

Reply:
Implemented one approach in https://github.com/cvmfs/cvmfs/pull/2863/commits/03bbc163cf6e41cb8ecec8574dd4fb0263f244fd